lib/restate-constructs/restate-cloud-environment.ts

```diff
@@ -56,8 +62,9 @@ export class RestateCloudEnvironment extends Construct implements IRestateEnviro
     super(scope, id);
     this.invokerRole = this.createInvokerRole(this, props);
     this.authToken = props.apiKey;
-    this.adminUrl = adminEndpoint(RESTATE_CLOUD_REGION_US, props.environmentId);
-    this.ingressUrl = ingressEndpoint(RESTATE_CLOUD_REGION_US, props.environmentId);
+    this.region = props.region ?? RESTATE_CLOUD_REGION_US;
```

Comment:
This should happen before the `invokerRole` assignment above :-) I'm going to fix it on a new PR since I can't push to your fork.

Reply:
:facepalm: